2023-07-21 10:17:50,526 INFO    MainThread:28091 [internal.py:wandb_internal():90] W&B internal server running at pid: 28091, started at: 2023-07-21 10:17:50.525954
2023-07-21 10:17:50,527 DEBUG   HandlerThread:28091 [handler.py:handle_request():141] handle_request: run_start
2023-07-21 10:17:50,528 INFO    WriterThread:28091 [datastore.py:open_for_write():75] open: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230721_101750-ub6ri6ns/run-ub6ri6ns.wandb
2023-07-21 10:17:50,529 DEBUG   HandlerThread:28091 [meta.py:__init__():35] meta init
2023-07-21 10:17:50,530 DEBUG   HandlerThread:28091 [meta.py:__init__():49] meta init done
2023-07-21 10:17:50,530 DEBUG   HandlerThread:28091 [meta.py:probe():209] probe
2023-07-21 10:17:50,531 DEBUG   HandlerThread:28091 [meta.py:_setup_git():199] setup git
2023-07-21 10:17:50,538 DEBUG   HandlerThread:28091 [meta.py:_setup_git():206] setup git done
2023-07-21 10:17:50,538 DEBUG   HandlerThread:28091 [meta.py:_save_pip():53] save pip
2023-07-21 10:17:50,538 DEBUG   HandlerThread:28091 [meta.py:_save_pip():67] save pip done
2023-07-21 10:17:50,538 DEBUG   HandlerThread:28091 [meta.py:probe():247] probe done
2023-07-21 10:17:51,756 WARNING MainThread:28091 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-07-21 10:17:51,922 WARNING MainThread:28091 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-07-21 10:17:51,923 ERROR   MainThread:28091 [internal.py:wandb_internal():156] Internal process interrupted.
2023-07-21 10:17:52,530 INFO    SenderThread:28091 [sender.py:finish():1111] shutting down sender
2023-07-21 10:17:52,857 INFO    WriterThread:28091 [datastore.py:close():279] close: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230721_101750-ub6ri6ns/run-ub6ri6ns.wandb
2023-07-21 10:17:52,858 INFO    HandlerThread:28091 [handler.py:finish():806] shutting down handler
2023-07-21 10:17:52,859 INFO    MainThread:28091 [internal.py:handle_exit():80] Internal process exited
